src/University/Teacher.java

```diff
@@ -0,0 +1,16 @@
+package University;
+
+public  class Teacher extends Person {
+    void printSalary( int rate, int  premium){
+        System.out.println("The salary of this teacher : " + (rate+premium));
+    }
+    void printSalary(int rate){
+        System.out.println("The salary of this teacher : " + rate);
+    }
+    static void printSalary(int rate, int hours, int day){
```

Comment:
this methods params should be as part of Teacher or Person, read some article about oop design
@ruslanguliarenko please make some consultation in details about requirements for @kuchmei 
We spend too much time on nothing